conformance/tests/dataclasses_transform_func.py

```diff
@@ -35,17 +35,28 @@ class Customer1:
     id: int
     name: str
 
+    def __init__(self, id: int, name: str) -> None:
```

Comment:
Adding an `__init__` in this case is not needed, and it incorrectly changes the nature of this test. This class is supposed to be a dataclass-like class, so its instance methods are initialized via a synthesized `__init__` method. If you add an explicit `__init__` method, this will cause the dataclass mechanism not to synthesize an `__init__`.